diff --git a/hentaifetch b/hentaifetch
index 25ae38f..981f544 100755
--- a/hentaifetch
+++ b/hentaifetch
@@ -1,6 +1,7 @@
 #!/bin/sh
 
 tmpfile="$(mktemp)"
+tmpascii="$(mktemp)"
 
 # Fetches wholesomelist for a random h*
 page=$(curl -fsSL "https://wholesomelist.com/random")
@@ -15,9 +16,11 @@ id=$(echo "$link" | grep -m1 -o "[[:digit:]]*")
 # fetches the image into a temporary file
 curl -fsSL "$img" > "$tmpfile.jpg"
 
+jp2a "$tmpfile.jpg" > "$tmpascii"
+
 # Displays the image as ascii with jp2a backend in neofetch
-neofetch --jp2a "$tmpfile.jpg"
+neofetch --ascii "$tmpascii"
 printf "[36m[1mNHentai tag: [0m#$id\n" # Show the nhentai numbers for research purposes
 
 # Delete the temporary file
-rm "$tmpfile" "$tmpfile.jpg"
\ No newline at end of file
+rm "$tmpfile" "$tmpfile.jpg" "$tmpascii"
